kuksa-client/kuksa_client/__main__.py

```diff
@@ -111,11 +174,16 @@ def path_completer(self, text, line, begidx, endidx):
                 description = ""
                 nodetype = "unknown"
 
-                if 'description' in child:
-                    description = child['description']
-
-                if 'type' in child:
-                    nodetype = child['type'].capitalize()
+                if self.serverProtocol == "grpc":
```

Comment:
I guess the problem that Sebastian reports is caused by that you do `child = childTree[key]` on line 172, and then `if 'description' in child[key]` on the line below.

In the case of Sebastian key seems to be `Vehicle`, but `childTree['Vehicle']['Vehicle']` does not seem to exist.

Reply:
Yes you are right, have to change it

Reply:
Problem fixed, the vss tree leafs children were wrongly created, changes were uploaded